pom.xml

```diff
@@ -990,9 +990,9 @@
           <version>1.5.1</version>
           <configuration>
             <printEqualFiles>false</printEqualFiles>
-            <failBuildInCaseOfDifferentContentConflict>false</failBuildInCaseOfDifferentContentConflict>
+            <failBuildInCaseOfDifferentContentConflict>true</failBuildInCaseOfDifferentContentConflict>
             <failBuildInCaseOfEqualContentConflict>false</failBuildInCaseOfEqualContentConflict>
-            <failBuildInCaseOfConflict>false</failBuildInCaseOfConflict>
+            <failBuildInCaseOfConflict>true</failBuildInCaseOfConflict>
```

Comment:
`failBuildInCaseOfConflict` is a superset to the previous two options, but it completely ignores the content checksum.   We should specify only the two independent complimentary options, since `failBuildInCaseOfConflict -> true` stomps on them and still causes build failures if we comment out the `about.html` exclusion you added earlier

```suggestion
```